qiskit_metal/renderers/renderer_mpl/patch.py

```diff
@@ -0,0 +1,114 @@
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+# ###########################################################################*/
+"""Paths and patches"""
+
+from matplotlib.patches import PathPatch
+from matplotlib.path import Path
+from numpy import asarray, concatenate, ones
+
+
+class Polygon(object):
+    """Adapt Shapely or GeoJSON/geo_interface polygons to a common interface"""
+
+    def __init__(self, context):
+        if hasattr(context, 'interiors'):
+            self.context = context
+        else:
+            self.context = getattr(context, '__geo_interface__', context)
+
+    @property
+    def geom_type(self):
+        return (getattr(self.context, 'geom_type', None) or
+                self.context['type'])
+
+    @property
+    def exterior(self):
+        return (getattr(self.context, 'exterior', None) or
+                self.context['coordinates'][0])
+
+    @property
+    def interiors(self):
+        value = getattr(self.context, 'interiors', None)
+        if value is None:
+            value = self.context['coordinates'][1:]
+        return value
+
+
+def PolygonPath(polygon: Polygon):
+    """Constructs a compound matplotlib path from a Shapely or GeoJSON-like
+    geometric object
+
+    Args:
+        polygon (Polygon): polygon
+
+    Returns:
+        Path: compound matplotlib path
+    """
+    this = Polygon(polygon)
+    assert this.geom_type == 'Polygon'
+
+    def coding(ob):
+        # The codes will be all "LINETO" commands, except for "MOVETO"s at the
+        # beginning of each subpath
+        n = len(getattr(ob, 'coords', None) or ob)
+        vals = ones(n, dtype=Path.code_type) * Path.LINETO
+        vals[0] = Path.MOVETO
+        return vals
+
+    vertices = concatenate([asarray(this.exterior.coords)[:, :2]] +
+                           [asarray(r.coords)[:, :2] for r in this.interiors])
+    codes = concatenate([coding(this.exterior)] +
+                        [coding(r) for r in this.interiors])
+    return Path(vertices, codes)
+
+
+def PolygonPatch(polygon: Polygon, **kwargs):
+    """Constructs a matplotlib patch from a geometric object
+    
```

Comment:
I think the line with Args: is needed.  
Args:

Reply:
It may help to indent the text under Args:

Reply:
Ok, I added the args for this function.